addons/im_livechat/static/src/legacy/public_livechat_chatbot.js

```diff
@@ -426,9 +426,9 @@ const QWeb = core.qweb;
             } else if (
                 (this._chatbotIsExpectingUserInput() ||
                 this.messaging.publicLivechatGlobal.livechatButtonView.chatbot.currentStep.data.chatbot_step_type === 'question_selection') &&
-                this.messaging.publicLivechatGlobal.livechatButtonView.messages.length !== 0
+                this.messaging.publicLivechatGlobal.messages.length !== 0
             ) {
-                const lastMessage = this.messaging.publicLivechatGlobal.livechatButtonView.messages[this.messaging.publicLivechatGlobal.livechatButtonView.messages.length - 1];
+                const lastMessage = this.messaging.publicLivechatGlobal.lastMessage;
```

Comment:
useless variable?

Reply:
Kept for the moment, just because the boolean expression would be long without it.
Eventually this will be turned into a computed field on `PublicLivechatGlobal`, and we'll drop the variable at this moment as `this.messaging.publicLivechatGlobal.lastMessage` will just become `this.lastMessage`, making the variable very much useless.